docs/xxcheckpointgcthread.md

```diff
@@ -0,0 +1,61 @@
+<!--
+* Copyright (c) 2017, 2023 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:CheckpointGCThreads
+
+**(Linux&reg; x86, Linux on POWER&reg; (Little Endian), and Linux on IBM Z&reg; only)**
+
+At the VM startup, you can set the number of threads that the garbage collector uses for parallel operations ([`-Xgcthreads`](xgcthreads.md)). At the time of taking the checkpoint, the `-XX:CheckpointGCThreads` option reduces the number of garbage collection (GC) threads that was set at startup.
+
+:fontawesome-solid-triangle-exclamation:{: .warn aria-hidden="true"} **Restrictions:** This option takes effect only when `-XX:+EnableCRIUSupport` is enabled.
+
+## Syntax
+
+        -XX:CheckpointGCThreads=<number>
+
+| Setting               | Value                            | Default                                                  |
+|-----------------------|-----------------------------------|:---------------------------------------------------------:|
+| `<number>` | Greater than 0 and less than or equal to the number of GC threads at the VM startup |  4           |
+
+If the `<number>` is greater than the number of GC threads at startup, the checkpoint fails with a `JVMCheckpoint` exception.
```

Comment:
The option is ignored in this case, the checkpoint will not fail.

Reply:
Done.